flocx_market/cmd/dbsync.py

```diff
@@ -11,3 +11,7 @@ def main():
     with application.app_context():
         orm.init_app(application)
         orm.create_all()
```

Comment:
I have moved this whole code block to app.py as well. Otherwise we run into the same issue when trying to run dbsynch. However, I'm not sure this is the best solution as then this service does nothing the api service doesnt already do

Reply:
Doing so still does not fix dbsynch, but it resolves the keystone side of it

Reply:
You don't need this at all; I'd take it out, and remove the dependencies on flask-keystone and flask-oslolog. The dbsync fix is in https://github.com/CCI-MOC/flocx-market/pull/77

Reply:
So I will leave this commit as is and let the other pr worry about dbsynch?

Reply:
Yeah!